mixer/cmd/reset.go

```diff
@@ -0,0 +1,200 @@
+// Copyright © 2018 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	"github.com/clearlinux/mixer-tools/builder"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var resetCmd = &cobra.Command{
+	Use:   "reset",
+	Short: "Revert the mix to a good state",
+	Long: `Revert the mix to a good state in case of a build 
+	failure or in case the user wants to roll back to a previous 
```

Comment:
I believe it is better to tell what the command does rather than telling when it should be used.